functions/oauth/authorize.ts

```diff
@@ -73,11 +76,11 @@ export async function handleRequestPost(
 	if (person === null) {
 		url.pathname = '/first-login'
 		url.searchParams.set('email', identity.email)
-		url.searchParams.set('redirect_uri', encodeURIComponent(redirect_uri + '?code=' + code))
+		url.searchParams.set('redirect_uri', encodeURIComponent(redirect_uri + '?code=' + code + state))
 		return URLsafeRedirect(url.toString())
 	}
 
-	return URLsafeRedirect(redirect_uri + '?code=' + code)
+	return URLsafeRedirect(redirect_uri + '?code=' + code + state)
```

Comment:
we've got two `redirect_uri + '?code=' + code + state`, wouldn't it be better to define a variable on line 74 for it and reuse it in both places? 

like 
```
const redirectUriParam = `redirect_uri?code=${code}${state}`
```
?

Reply:
Checked